src/Core/Output/AbsynCodeFormatter.cs

```diff
@@ -100,13 +100,15 @@ private string UnsignedRepresentation(Constant u)
             else
             {
                 var sb = new StringBuilder();
+#if USE_TILDA_IF_SMALLER                
```

Comment:
This looks like a user preference, and it shouldn't be coded using `#if` / `#endif`. Instead consider opening an issue to add support for changing the formatting of unsigned numbers, controlled by user preference. There we can discuss how this would best be accomplished.